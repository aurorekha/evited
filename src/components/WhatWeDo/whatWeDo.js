import React, { Component } from 'react';
import './whatWeDo.scss';

class WhatWeDo extends React.Component{
  render() {
    return (
      <div className="whatWeDo">
        <h1>What do we do?</h1>
        <p>We make awesome e-invites that can cater for all of your events.</p>
        <p>Sure, there are many apps that you can get on your phone or
          you can do it online but how can you personalize the invite for you event?</p>
        <p>Previously physical cards were given a lot of importance to and they used to cost a lot.<br /> Even now people go on
        to spend a lot on these cards but no more. We can do this while reducing carbon footprint and having fun!</p>
      </div>
    )
  }
}

export default WhatWeDo;
