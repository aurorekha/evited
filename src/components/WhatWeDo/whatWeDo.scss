@font-face {
  font-family: "arapey";
  src: url("../../fonts/arapey/arapey-italic-webfont.woff") format("woff"),
}

.whatWeDo {
  font-family: "arapey";
  background: #9AD2C2;
  color: white;
  padding: 2rem;
  h1 {
    padding: 2rem 0;
  }
  p {
    font-size: 1.5rem;
  }
}
