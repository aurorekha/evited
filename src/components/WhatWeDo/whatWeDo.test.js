import React from 'react';
import WhatWeDo from './whatWeDo';
import renderer from 'react-test-renderer';

it('renders WhatWeDo section correctly', () => {
  const tree = renderer.create(
    <WhatWeDo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
