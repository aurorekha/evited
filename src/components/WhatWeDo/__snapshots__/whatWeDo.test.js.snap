// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WhatWeDo section correctly 1`] = `
<div
  className="whatWeDo"
>
  <h1>
    What do we do?
  </h1>
  <p>
    We make awesome e-invites that can cater for all of your events.
  </p>
  <p>
    Sure, there are many apps that you can get on your phone or you can do it online but how can you personalize the invite for you event?
  </p>
  <p>
    Previously physical cards were given a lot of importance to and they used to cost a lot.
    <br />
     Even now people go on to spend a lot on these cards but no more. We can do this while reducing carbon footprint and having fun!
  </p>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="whatWeDo"
>
  <h1>
    What do we do?
  </h1>
  <p>
    We make awesome e-invites that can cater for all of your events.
  </p>
  <p>
    Sure, there are many apps that you can get on your phone or you can do it online but how can you personalize the invite for you event?
  </p>
  <p>
    Previously physical cards were given a lot of importance to and they used to cost a lot.
    <br />
     Even now people go on to spend a lot on these cards but no more. We can do this while reducing carbon footprint and having fun!
  </p>
</div>
`;
